main.py

```diff
@@ -1060,24 +1214,39 @@ def main():
 
             elif packet.is_type(SMSG_TRADE_COMPLETE):
                 commitMessage=""
-                # The sale_tree is only ammended after a complete trade packet.
+                # we get here which tree it should be through UID
+                if trader_state.item.get == 1:
```

Comment:
Was the error returned. 
Traceback (most recent call last):
  File "main.py", line 1287, in <module>
    main()
  File "main.py", line 1218, in main
    if trader_state.item.get == 1:
AttributeError: 'int' object has no attribute 'get'

Then most likely something isn't setting trader_state.item right or return it weird.